kustomize/data-plane/shard/resources/operator-config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: smart-events-shard-operator-config
+  namespace: openshift-operators
+data:
+  EVENT_BRIDGE_ISTIO_GATEWAY_NAME: istio-ingressgateway
```

Comment:
@winklerm You're just re-using the default values of `EVENT_BRIDGE_ISTIO_GATEWAY_NAME` , `EVENT_BRIDGE_ISTIO_GATEWAY_NAMESPACE` and `EVENT_BRIDGE_K8S_PLATFORM`.

Reply:
Yes, I was not sure if they were required to be in the configmap, probably not, thanks!